### DISCUSSION TITLE
publish to JSR

### DISCUSSION THREAD

**Discussion by Julian-B90** [stephanrauh/ngx-extended-pdf-viewer#2495] [Ideas]
Maybe we can publish this nice package to https://jsr.io ?

**Comment by stephanrauh** [comment]
It's the first time I hear about JSR. Is it a platform with major impact? What are the advantages of publishing on JSR?

**Reply by Julian-B90** [reply]
- It has Native TypeScript support
- Automatic API documentation generation from source code
- better search than [npm](https://npmjs.com/)

And there is a score for each package that shows how good the documentation is.

And actually as a direct replacement for NPM.

https://jsr.io/docs/why
